viz-lib/src/visualizations/table/renderer.less

```diff
@@ -66,6 +66,14 @@
           }
         }
 
+        &.table-fixed-header {
+          position: sticky;
+          left: 0;
+          top: 0;
+          border-top: hidden;
+          background: #fafafa !important;
```

Comment:
review this code, if didn't add `!important`, the table header will show this:
![image](https://user-images.githubusercontent.com/46274612/90310241-7ef3a400-df22-11ea-8a1e-2ceb37d98aed.png)
It will be hover override.

Reply:
NP, I was actually gonna ask you the reason for both that and the `z-index`.

Reply:
Haha...your NP means Chinese good? Thanks! At first i added `z-index` because of my wrong idea, now i want to correct this. I didn't test all feature, it's my fault.

Reply:
Sorry, NP = No Problems.

Don't worry, I already removed the `z-index`. Considering this is something we wanted for quite a long time, I'll just try to refine the border myself tomorrow (it has a 1 px border that doesn't follow the `sticky`) and after some more tests it should be ready to go 🙂.

Reply:
> 抱歉，NP =没问题。
> 
> 不用担心，我已经删除了`z-index`。考虑到这是我们长期以来一直想要的，我明天将尝试自己完善边框（边框有一个1 px的边框，该边框不遵循`sticky`），经过更多测试后，应该可以开始使用🙂。

Ok, if you perfect this feature, tell me, i want to add in my online redash. Thank you.

Reply:
> > Sorry, NP = No Problems.
> > Don't worry, I already removed the z-index. Considering this is something we wanted for quite a long time, I'll just try to refine the border myself tomorrow (it has a 1 px border that doesn't follow the sticky) and after some more tests it should be ready to go 🙂.
> 
> Ok, if you perfect this feature, tell me, i want to add in my online redash. Thank you.